go-controller/pkg/util/pod_annotation_unit_test.go

```diff
@@ -0,0 +1,266 @@
+package util
+
+import (
+	"fmt"
+	"net"
+	"reflect"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	ovntest "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/testing"
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/core/v1"
+)
+
+func TestMarshalPodAnnotation(t *testing.T) {
+	tests := []struct {
+		desc           string
+		inpPodAnnot    PodAnnotation
+		errAssert      bool  // used when an error string CANNOT be matched or sub-matched
+		errMatch       error //used when an error string CAN be matched or sub-matched
+		expectedOutput map[string]string
+	}{
+		{
+			desc:           "PodAnnotation instance with no fields set",
```

Comment:
you aligned the fields and values here, but in the subsequent entries they are not aligned

Reply:
gofmt when using the IDE automatically takes care of aligning them appropriately.
When I try to manually re-align them as can be seen below, I get an error when I execute `make gofmt`. This is because on Line31, a struct is being populated on mutli line

![Screen Shot 2020-07-13 at 5 10 59 PM](https://user-images.githubusercontent.com/62150532/87358763-146ad500-c52c-11ea-87e1-a568f87d85a3.png)